docs/src/main/sphinx/functions/table.rst

```diff
@@ -47,6 +47,12 @@ by the specified catalog, the query fails.
 The table function name is resolved case-insensitive, analogically to scalar
 function and table resolution in Trino.
 
+.. note::
+
+    Polymorphic table functions may not preserve the order of the function's
+    result, so an ``ORDER BY`` clause in a table function may not sort rows as
+    expected.
```

Comment:
Such a note would be more appropriate at `query` query pass-through function.

here, we can also have this information, but the wording would need to be quite different.... and users would still not see this, when looking at the `query` docs

Reply:
Agreed .. we have to come up with a larger strategy around this and other limitations we need to mention for query pass through PTF ... probably as an included fragment .. problem is that we did not even name the query pass through consistently across connectors...

Reply:
Question, how would the wording need to be different between discussing the limitation here and discussing it in the `query` function context? @findepi